README.md

```diff
@@ -277,6 +278,24 @@ Adds a `<Plane />` that always faces the camera.
 />
 ```
 
+#### GizmoHelper
+
+Used by widgets that visualize and control camera position.
+
+Two example gizmos are included: GizmoViewport and GizmoViewcube, and `useGizmoContext` makes it easy to create your own.
+
+```jsx
+<GizmoHelper
+  alignment="bottom-right" // widget alignment within scene
+  margin={[80, 80]} // widget margins (X, Y)
+  onUpdate={/* called during camera animation  */}
+  onTarget={/* return current camera target (e.g. from orbit controls) to center animation */}
+> 
+  <GizmoViewport />
+  {/* alternative: <GizmoViewcube /> */}
```

Comment:
can you add the props this takes to here please?

Reply:
sure thing!